pkg/jx/cmd/import.go

```diff
@@ -900,11 +900,18 @@ func (options *ImportOptions) doImport() error {
 		jenkinsfile = defaultJenkinsfileName
 	}
 
-	err = options.ensureDockerRepositoryExists()
+	dockerfileExists, err := util.FileExists("Dockerfile")
```

Comment:
ooi, are you, in this case, using the import with the `--no-draft=true` option? Otherwise you get a `Dockerfile`, right?

Say you add a `Dockerfile` later, you would expect that the repo is then manually created?

Reply:
Yes, I'm using `--no-draft=true`.
If I added a Dockerfile later, then yes I would manually create the repo.